README.md

```diff
@@ -51,15 +53,16 @@ If there isn't a download for your platform (see the downloads section above) yo
 7. Change directory to where the software files have been extracted
 8. Type and run `python mindfulness-at-the-computer.py` on Windows or `python3 mindfulness-at-the-computer.py`on GNU/Linux systems
 
-On MacOS you also need to install growl for the breathing reminder notifications to be shown
+NOTE:
+If you are on the MacOS, you need to install growl to get the breathing reminder notifications.
```

Comment:
@SunyataZero Because there is a **bolded** section for GNU/Linux, perhaps a **bolded** section for MacOS would be suitable as well.

Reply:
@andy5995 Thank you for the comment, i'm not sure, do you mean the headings? MacOS isn't really supported at the moment so i don't know, but maybe we can write all OSes in bold?